lib/octokit/error.rb

```diff
@@ -225,10 +223,10 @@ class Unauthorized < ClientError; end
   # Raised when GitHub returns a 401 HTTP status code
   # and headers include "X-GitHub-OTP"
   class OneTimePasswordRequired < ClientError
```

Comment:
I don't think this code - or anything else around OTPs - is relevant with modern GitHub authentication methods.